emData/generate_ME.py

```diff
@@ -75,15 +75,19 @@ def writeModuleToImplementation(topFile, funcName, prefix, layerDisk, label):
         "#ifdef  __VITIS_HLS__\n"
         "\t// For use with Vitis >=2020.1\n"
         "\t#pragma HLS LATENCY min=1 max=1\n"
-        "\t#pragma HLS interface ap_memory port=inputStubData->get_mem() latency=2\n"
-        "\t#pragma HLS interface ap_memory port=inputProjectionData->get_mem() latency=2\n"
+        "\t#pragma HLS interface ap_memory port=inputStubData.get_mem() latency=2\n"
+        "\t#pragma HLS interface ap_memory port=inputProjectionData.get_mem() latency=2\n"
         "#else\n"
         "\t#pragma HLS interface register port=bx_o\n"
         "\t#pragma HLS resource variable=inputStubData->get_mem() latency=2\n"
         "\t#pragma HLS resource variable=inputProjectionData->get_mem() latency=2\n"
         "#endif\n"
         "\n"
+	"#ifdef  __VITIS_HLS__\n"
```

Comment:
Why is this #ifdef needed? The code on either side of the #else appears identical.